packages/block-editor/src/components/url-popover/link-viewer.js

```diff
@@ -7,12 +7,9 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { pencil } from '@wordpress/icons';
-import { createSlotFill, ExternalLink, Button } from '@wordpress/components';
+import { ExternalLink, Button } from '@wordpress/components';
 import { safeDecodeURI, filterURLForDisplay } from '@wordpress/url';
-import { useMemo } from '@wordpress/element';
-
-const { Slot, Fill } = createSlotFill( 'BlockEditorURLPopoverLinkViewer' );
```

Comment:
If the `URLPopover` has been replaced, why do we need to make changes to this file?

Could there be any cases where this is still used where removing this slot may cause issues?

Reply:
This was only added for this, so I am reverting it here before anyone gets a chance to use it.